src/shared/creds-util.c

```diff
@@ -1033,6 +1035,10 @@ int decrypt_credential_and_warn(
                                     le32toh(z->size));
                 }
 
+                /*
+                 * TODO: Add the SRK data to the credential structure so it can be plumbed
+                 * through and used to verify the TPM session.
+                 */
```

Comment:
hmm, so we have this rule that we use `/* */` for explanatory comments, and `//` for "fixme" style comments. This is one of the latter, hence should use //. but doesn't matter much